src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -1040,19 +1040,19 @@ private static bool TryGetArg0AsDsInfo(CallNode callNode, TexlBinding binding, o
             return result;
         }
 
-        protected bool SetErrorForMismatchedColumns(DType expectedType, DType actualType, TexlNode errorArg, IErrorContainer errors, Features features)
+        internal static bool SetErrorForMismatchedColumns(DType expectedType, DType actualType, TexlNode errorArg, IErrorContainer errors, Features features, bool requireAllParamColumns)
```

Comment:
>requireAllParamColumns [](http://example.com/codeflow?start=163&length=22)

Consider naming it something like `requireAllParamColumnsOverride` to avoid confusion with the instance property of the same name (just different casing for the first letter)